src/store/store.ts

```diff
@@ -1114,6 +1114,7 @@ export const useStore = defineStore('store', {
     /** True if resolution dates are valid. */
     getIsResolutionDatesValid (): boolean {
       if (
+        this.isAlterationFiling &&
```

Comment:
This is the main fix: only check validity if this is an alteration filing (ie, not for a correction filing).